### ISSUE TITLE
Users - Invited user status message not announced

### ISSUE THREAD

**Issue by katknow** [omeka/Omeka#1143]
"Description: Upon inviting a user to the Omeka site, there is a status message that appears at the top of the page announcing that this user has been invited to join your Omeka site. This status message is not announced to screen readers. Messages of this kind should be announced automatically after the change has been made or a submit button has been pressed.

Note: This also occurs when entering an invalid email address in the edit field.

Expected Results: All status messages that appear onscreen are automatically announced.

Actual Results: Status messages are not announced."

4.1.3 Status Messages; Level AA; WCAG 2.1
